tests/matrix_free/matrix_vector_faces_26.output.skylake

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
Please rename into `.output.avx512` to be consistent with other files, e.g., `tests/grid/mesh_3d_12.output.avx512`.